services/src/handlers/tasks.rs

```diff
@@ -0,0 +1,271 @@
+use crate::error::Result;
+use crate::tasks::{TaskDb, TaskListOptions};
+use crate::{contexts::Context, tasks::TaskId};
+use actix_web::{web, FromRequest, Responder};
+use serde::{Deserialize, Serialize};
+
+pub(crate) fn init_task_routes<C>(cfg: &mut web::ServiceConfig)
+where
+    C: Context,
+    C::Session: FromRequest,
+{
+    cfg.service(
+        web::scope("/tasks")
+            .service(web::resource("/list").route(web::get().to(list_handler::<C>)))
+            .service(web::resource("/{task_id}/status").route(web::get().to(status_handler::<C>))),
+    );
+}
+
+/// Create a task somewhere and respond with a task id and a url to query the task status.
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct TaskResponse {
+    pub task_id: TaskId,
+    pub url: String,
```

Comment:
I don't like this idea of returning URLs. We do it nowhere else, why here? e.g. we don't create a project and return a URL to access it.

Reply:
[ccd3657](https://github.com/geo-engine/geoengine/pull/555/commits/ccd3657e069e1c575890b2033728e91934dc0657)